src/main/java/com/geekbeast/rhizome/pods/hazelcast/RegistryBasedHazelcastInstanceConfigurationPod.java

```diff
@@ -45,17 +44,23 @@
 public class RegistryBasedHazelcastInstanceConfigurationPod {
     private static final Logger                                               logger             = LoggerFactory
                                                                                                          .getLogger( RegistryBasedHazelcastInstanceConfigurationPod.class );
-    private static final ConcurrentMap<Class<?>, Serializer>                  serializerRegistry = Maps.newConcurrentMap();
-    private static final ConcurrentMap<String, SelfRegisteringMapStore<?, ?>> mapRegistry        = Maps.newConcurrentMap();
-    private static final ConcurrentMap<String, SelfRegisteringQueueStore<?>>  queueRegistry      = Maps.newConcurrentMap();
-    private static final ConcurrentMap<Class<?>, KeyMapper<?>>                keyMapperRegistry = Maps.newConcurrentMap();
+    private static final ConcurrentMap<Class<?>, Serializer>                  serializerRegistry = Maps
+                                                                                                         .newConcurrentMap();
```

Comment:
I think Matt's formatter is off or something. Mine are all unchanged. In fact this file actually doesn't have any changes except formatting.